ref/app-architecture.md

```diff
@@ -1,11 +1,13 @@
 # App Architecture
 
+The Top Cat App will use an Entity/Hydrator design pattern to pull data from a datbase and create an array of objects of the same class
```

Comment:
Misspelling: `datbase` -> `database`.
Could you modify `objects of the same class` to `CatEntity objects`? The more precise the better!